src/react/atlascode/config/auth/CloudAuthButton.tsx

```diff
@@ -18,7 +18,13 @@ export const CloudAuthButton: React.FunctionComponent<CloudAuthButtonProps> = ({
     };
 
     return (
-        <Button variant="contained" color="primary" startIcon={<CloudIcon />} onClick={() => handleCloudProd()}>
+        <Button
+            id="settings.login.cloud.button"
```

Comment:
NIT: I'd strongly recommend using `kebab-case` for element ids because it is in line with other `id` values in the repo - as well as best practices, to my knowledge.

TIL dots were even allowed in `id`s , seeing them kind of breaks my brain a little bit because of selector syntax where `#a.b.c` means "Element with id `a` that has classes `b` and `c`" - [relevant example](https://css-tricks.com/multiple-class-id-selectors/)

Reply:
Ahhh okay. Will fix up in a follow up PR.